packages/jbrowse-plugin-apollo/src/ApolloRenderer/components/ApolloRendering.tsx

```diff
@@ -282,6 +296,16 @@ function ApolloRendering(props: ApolloRenderingProps) {
         bp: apolloFeatureUnderMouse[overEdge],
       })
     }
+    if (apolloFeatureUnderMouse && internetAccount) {
+      const userLocation = {
+        assemblyId: apolloFeatureUnderMouse.assemblyId,
+        refSeq: apolloFeatureUnderMouse.refSeq,
+        featureId: apolloFeatureUnderMouse._id,
```

Comment:
We don't need a feature ID, only a location.

Reply:
Unnecessary featureId removed.